models/mednist_gan/configs/train.json

```diff
@@ -0,0 +1,168 @@
+{
+    "imports": [
+        "$from functools import partial",
+        "$import glob",
+        "$import numpy as np",
+        "$import torch",
```

Comment:
No need to import numpy and torch as they are dependent packages of MONAI.

Thanks.

Reply:
Sorted.